### ISSUE TITLE
[Docs] Observability of CREATE INDEX waiting on backends

### ISSUE THREAD

**Issue by jasonyb** [yugabyte/yugabyte-db#17272]
### Description

CREATE INDEX now can be "stuck" waiting on backends to be resolved, so documentation for diagnosing that should be added.  This may want to be postponed until the feature is present in a stable release.

See commit a1729c352896e919f462c614770da443b9982c0a.

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.
